src/content/1.7/development/webservice/reference.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Reference
+weight: 3
+---
+
+# Web service reference
+
+All webservice APIs are accessible through the `/api/` gateway. For instance, `http://example.com/api/carriers`
+
+## Available HTTP methods
+
+Most resources can be accessed in a REST manner, with the 5 main HTTP request methods: GET, POST, PUT, DELETE, HEAD. The only exceptions are:
+
+- `search`: only GET and HEAD.
+- `stock_availables`: only GET, POST, and HEAD.
+- `stock_movements`: only GET and HEAD.
+- `stocks`: only GET and HEAD.
+- `supply_order_details`: only GET and HEAD.
+- `supply_order_histories`: only GET and HEAD.
+- `supply_order_receipt_histories`: only GET and HEAD.
+- `supply_order_states`: only GET and HEAD.
+- `supply_orders`: only GET and HEAD.
+- `warehouse_product_locations`: only GET and HEAD.
+- `warehouses`: only GET, POST, PUT, and HEAD.
```

Comment:
Consider using a compatibility table, like this

 /  | GET | POST | PUT | HEAD
---| :---: | :---: | :---: | :---:
search | X |    |     | X